static/app/views/dashboardsV2/widgetCard/index.tsx

```diff
@@ -282,41 +280,78 @@ class WidgetCard extends Component<Props, State> {
                   windowWidth={windowWidth}
                   onDataFetched={this.setData}
                 />
-              </LazyLoad>
-            )}
-            {this.renderToolbar()}
-          </WidgetCardPanel>
-          <Feature organization={organization} features={['dashboards-mep']}>
-            <DashboardsMEPConsumer>
-              {({isMetricsData}) =>
-                showStoredAlert &&
-                widget.widgetType === WidgetType.DISCOVER &&
-                isMetricsData === false && (
-                  <StoredDataAlert showIcon>
-                    {tct(
-                      "Your selection is only applicable to [storedData: stored event data]. We've automatically adjusted your results.",
-                      {
-                        storedData: <ExternalLink href="https://docs.sentry.io/" />, // TODO(dashboards): Update the docs URL
-                      }
-                    )}
-                  </StoredDataAlert>
-                )
-              }
-            </DashboardsMEPConsumer>
-          </Feature>
-          <Feature organization={organization} features={['dashboards-releases']}>
-            {showIncompleteDataAlert && (
-              <StoredDataAlert showIcon>
-                {tct(
-                  'Releases data is only available from [date]. Data may be incomplete as a result.',
-                  {
-                    date: <DateTime date={METRICS_BACKED_SESSIONS_START_DATE} dateOnly />,
+              ) : (
+                <LazyLoad once resize height={200}>
+                  <WidgetCardChartContainer
+                    api={api}
+                    organization={organization}
+                    selection={selection}
+                    widget={widget}
+                    isMobile={isMobile}
+                    renderErrorMessage={renderErrorMessage}
+                    tableItemLimit={tableItemLimit}
+                    windowWidth={windowWidth}
+                    onDataFetched={this.setData}
+                  />
+                </LazyLoad>
+              )}
+              {this.renderToolbar()}
+            </WidgetCardPanel>
+            <Feature organization={organization} features={['dashboards-mep']}>
+              <DashboardsMEPConsumer>
+                {({isMetricsData}) => {
+                  if (
+                    isMetricsData === false &&
```

Comment:
there's a functional difference here because `isMetricsData` can be undefined if it hasnt been set by the provider yet.
When opening the widget builder, theres a slight moment in time when the events request is still inflight and `isMetricsData` is undefined because we rely on the result of the events request to set to value.
We want to check explicitly for false so we don't accidentally render the banner until we know if it's truly not a `isMetricsData` response.

Reply:
Ah, that makes sense. My bad, thanks for clarifying that. I forgot that it's set based off of the returned response.